### PR TITLE
Added config.ini, modularized exception handling.

### PR DESCRIPTION
**Pull request by chavarera**
ADDED:
- CODE OPTIMIZED
- COMMAND_HANDLER (contain all Handeler)
- New File Added (config.ini) to Store User TOKEN
- EXCEPTION HANDELR ADDED

### CODE REVIEW

File: tgbot.py [hunk 1]
```diff
@@ -7,9 +7,32 @@
...
+                        print("Token Should not be Empty please add it in config.ini file")
+                        return False
+                elif '"' in token or "'" in token:
+                        print("Do not add Quotes to token use direct value\ntoken=XXXX:XXXXXX")
+                        return False
+                        
+                else:
+                        return token
+        except:
+                print('''Do not change Syntax of config.ini file It should be Like\n[telegram]\ntoken="YOUR TOKEN"''')
```
Comment (nileshpatra) [inline tgbot.py]: change it to: `print('Do not change Syntax of config.ini file!')`

File: tgbot.py [hunk 2]
```diff
@@ -177,28 +200,33 @@ def filter_reply(bot , update):
...
+                                 'github':getlink,
+                                 'website':getlink,
+                                 'meetup':getlink}
+                for key,value in COMMAND_HANDLER.items():
+                        dp.add_handler(CommandHandler(key ,value))
+                dp.add_handler(MessageHandler(Filters.status_update, welcome_message))
+                updater.start_polling()
+                updater.idle()
+        except:
+                print("Token Is Invalid Please Use Correct Token key in config.ini")
```
Comment (nileshpatra) [inline tgbot.py]: change to: `print('The Token Is Invalid! Use Correct Token in config.ini file')`

File: config.ini [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[telegram]
+token=TOKEN_KEY_WITHOUT_QUOTES
+
```
Comment (nileshpatra) [inline config.ini]: Remove extra newlines

File: tgbot.py [hunk 4]
```diff
@@ -177,28 +200,33 @@ def filter_reply(bot , update):
...
-	dp.add_handler(CommandHandler('meetup', getlink))
-	dp.add_handler(MessageHandler(Filters.status_update, welcome_message))
-	updater.start_polling()
-	updater.idle()
-
+def main(TOKEN):
+        try:
+                updater = Updater(TOKEN)
+                dp = updater.dispatcher
+                COMMAND_HANDLER={'start':startmessage,
```
Comment (nileshpatra) [inline tgbot.py]: change the variable name from `COMMAND_HANDLER` to `BOT_COMMANDS`

**Comment by nileshpatra** [comment]
Hi @chavarera ! Thank you for the PR. It overall looks good to me. Just a few small nitpicks, and I have added them as comments. Just rectify them and I think this should be ready to merge o/

**Comment by chavarera** [comment]
@nileshpatra  ok i will modify it according to need

File: tgbot.py (hunk 1 above)
Comment (chavarera) [reply tgbot.py]: @nileshpatra Completed

File: tgbot.py (hunk 4 above)
Comment (chavarera) [reply tgbot.py]: @nileshpatra done

File: config.ini (hunk 3 above)
Comment (chavarera) [reply config.ini]: @nileshpatra  done

File: config.ini [hunk 5]
```diff
@@ -0,0 +1,2 @@
+[telegram]
+token=787219918:AAFNTjvxvaQDDHv6dCwJ9nBQUKKDDoeymIs
```
Comment (nileshpatra) [inline config.ini]: @chavarera you just pushed your token along!!

**Comment by chavarera** [comment]
@nileshpatra according to your comments i have modified it check it and if any issue occure inform me

File: tgbot.py [hunk 6]
```diff
@@ -177,28 +200,33 @@ def filter_reply(bot , update):
...
+                for key,value in BOT_COMMANDS.items():
+                        dp.add_handler(CommandHandler(key ,value))
+                dp.add_handler(MessageHandler(Filters.status_update, welcome_message))
+                updater.start_polling()
+                updater.idle()
+        except:
+                print("The Token Is Invalid! Use Correct Token in config.ini file")
 if __name__ == '__main__':
-	main()
+        TOKEN=CheckConfig()
```
Comment (nileshpatra) [inline tgbot.py]: To Keep uniform spacing could you change it to: ```TOKEN = Check config()```

**Comment by nileshpatra** [comment]
@chavarera added more comments, :)

**Comment by nileshpatra** [comment]
Also, it would be great if you could change the commit message to something like: Added config.ini, modularized exception handling. Also, please squash your commits into one commit.

File: config.ini (hunk 5 above)
Comment (nileshpatra) [reply config.ini]: Also, please add space when you fix this. ```token = YOUR_TOKEN_HERE```

**Pull request closed without merging**